src/logics/Logic.cc

```diff
@@ -956,6 +958,11 @@ PTRef Logic::mkVar(SRef s, const char* name) {
     return ptr;
 }
 
+PTRef Logic::mkUniqueVar(SRef s) {
+    std::string uniqueName = "@" + std::to_string(uniqueVarCount++);
```

Comment:
Put the "@" in some const char near line 65 and use that instead of hardcoding it in the function.

Reply:
Fixed.